sdk/typescript/src/dwallet-mpc/sign.ts

```diff
@@ -116,3 +131,213 @@ export async function sign(
+
+	const centralizedSignedMessage = create_sign_centralized_output(
+		networkDecryptionKeyPublicOutput,
+		MPCKeyScheme.Secp256k1,
+		activeDWallet.state.fields.public_output,
+		secretKey,
+		presign.presign,
+		message,
+		hash,
+	);
+	const dWalletStateData = await getDWalletSecpState(conf);
+	const tx = new Transaction();
+
+	const emptyIKACoin = tx.moveCall({
+		target: `${SUI_PACKAGE_ID}::coin::zero`,
+		arguments: [],
+		typeArguments: [`${conf.ikaConfig.ika_package_id}::ika::IKA`],
+	});
+
+	const [unverifiedECDSAPartialUserSignatureCap] = tx.moveCall({
+		target: `${conf.ikaConfig.ika_system_package_id}::${DWALLET_ECDSA_K1_MOVE_MODULE_NAME}::request_ecdsa_future_sign`,
+		arguments: [
+			tx.sharedObjectRef({
+				objectId: dWalletStateData.object_id,
+				initialSharedVersion: dWalletStateData.initial_shared_version,
+				mutable: true,
+			}),
+			tx.pure.id(dwalletID),
+			tx.pure(bcs.vector(bcs.u8()).serialize(message)),
+			tx.pure.id(presignID),
+			tx.pure(bcs.u8().serialize(hash.valueOf())),
+			tx.pure(bcs.vector(bcs.u8()).serialize(centralizedSignedMessage)),
+			emptyIKACoin,
+			tx.gas,
+		],
+	});
+	tx.transferObjects(
+		[unverifiedECDSAPartialUserSignatureCap],
+		conf.suiClientKeypair.toSuiAddress(),
+	);
+	tx.moveCall({
+		target: `${SUI_PACKAGE_ID}::coin::destroy_zero`,
+		arguments: [emptyIKACoin],
+		typeArguments: [`${conf.ikaConfig.ika_package_id}::ika::IKA`],
+	});
+	const result = await conf.client.signAndExecuteTransaction({
+		signer: conf.suiClientKeypair,
+		transaction: tx,
+		options: {
+			showEffects: true,
+			showEvents: true,
+			showObjectChanges: true,
+		},
+	});
+	const startSessionEvent = result.events?.at(0)?.parsedJson;
+	if (!isStartSessionEvent(startSessionEvent)) {
+		throw new Error('invalid start session event');
+	}
+
+	const completedSignEventType = `${conf.ikaConfig.ika_system_package_id}::${DWALLET_ECDSA_K1_INNER_MOVE_MODULE_NAME}::CompletedECDSAFutureSignEvent`;
+	await fetchCompletedEvent(
+		conf,
+		startSessionEvent.session_id,
+		isCompletedFutureSignEvent,
+		completedSignEventType,
+	);
+
+	const objects = result.objectChanges!;
+	if (!objects) {
+		throw new Error('no objects created');
+	}
+	for (const obj of objects) {
+		if (
+			obj &&
+			'objectType' in obj &&
+			obj.objectType! ===
+				`${conf.ikaConfig.ika_system_package_id}::${DWALLET_ECDSA_K1_INNER_MOVE_MODULE_NAME}::UnverifiedECDSAPartialUserSignatureCap`
+		) {
+			return obj.objectId;
+		}
+	}
+	return undefined;
+}
+
+export async function verifyECFSASignWithPartialUserSignatures(
+	conf: Config,
+	unverifiedECDSAPartialUserSignatureCapID: string,
+): Promise<string | undefined> {
+	const dWalletStateData = await getDWalletSecpState(conf);
+	const tx = new Transaction();
+
+	const [verifiedECDSAPartialUserSignatureCap] = tx.moveCall({
+		target: `${conf.ikaConfig.ika_system_package_id}::${DWALLET_ECDSA_K1_MOVE_MODULE_NAME}::verifiy_ecdsa_partial_user_signature_cap`,
```

Comment:
```suggestion
		target: `${conf.ikaConfig.ika_system_package_id}::${DWALLET_ECDSA_K1_MOVE_MODULE_NAME}::verify_ecdsa_partial_user_signature_cap`,
```
There is a typo in the func name, fix it, also fix in move if needed.